client/dashboard/.eslintrc.js

```diff
@@ -45,6 +45,7 @@ module.exports = {
 							'!@automattic/components/src/summary-button',
 							'!@automattic/components/src/breadcrumbs',
 							'!@automattic/components/src/breadcrumbs/types',
+							'!@automattic/components/src/gridicon',
```

Comment:
We're trying out keep gridicon out of the HD for as long as we can 🙂
In this case, we can use `plugins` from `@wordpress/icons`.

Reply:
It works, thanks!